projects/storefrontapp/tsconfig.server.json

```diff
@@ -402,6 +402,9 @@
       "@spartacus/cdc/core": ["../../integration-libs/cdc/core/public_api"],
       "@spartacus/cdc": ["../../integration-libs/cdc/public_api"],
       "@spartacus/cdc/root": ["../../integration-libs/cdc/root/public_api"],
+      "@spartacus/cdc/manage-users": [
+        "../../integration-libs/cdc/manage-users/public_api"
+      ],
```

Comment:
please move this to the last of this array.

Reply:
done